tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -103,6 +109,68 @@ var _ = Describe("odo devfile watch command tests", func() {
 		})
 	})
 
+	Context("when executing odo watch", func() {
+		It("should show validation errors if the devfile is incorrect", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, cmpName)
+
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(context, "devfile.yaml"))
+
+			output := helper.CmdShouldPass("odo", "push", "--project", namespace)
+			Expect(output).To(ContainSubstring("Changes successfully pushed to component"))
+
+			session := helper.CmdRunner("odo", "watch", "--context", context)
+			defer session.Kill()
```

Comment:
Do we need the `dever session.Kill()` here if we're calling `session.Kill()` down below?

Reply:
We don't need the session kill below, I've removed it in the appropriate spots.